### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shraddha761**
I tried to Improvised sentence in readme

### CODE REVIEW

**Comment by shraddha761** [comment]
@roberto-butti please review

**Comment by shraddha761** [comment]
@roberto-butti is there any updation required

File: README.md [hunk 1]
```diff
@@ -37,9 +37,9 @@
 
 I've put together a package of useful statistical functions.
 
-These functions originally stemmed from my exploration of FIT files, which contain a wealth of data about sports activities. Within these files, you can find detailed information on metrics such as Heart Rate, Speed, Cadence, Power, and more. I developed these statistical functions to help gain deeper insights into the numerical data and performance of these sports activities.
+These functions were initially conceived during my exploration of FIT files, which are rich sources of data pertaining to sports activities. Enclosed within these files, you'll discover in-depth information on metrics like Heart Rate, Speed, Cadence, Power, and more. I designed these statistical functions to facilitate a more profound understanding of the numerical data and the performance of these sports activities.
```
Comment (roberto-butti) [inline README.md:40]: Thank you for the feedback, suggestion, but I keep the prev version

File: README.md [hunk 2]
```diff
@@ -37,9 +37,9 @@
...
 
-The functions provided by this package, cover a range of measures, including mean, mode, median, range, quantiles, first quartile (25th percentile), third quartile (75th percentile), frequency tables (cumulative and relative), standard deviation (both for populations and samples), and variance (again for populations and samples).
+This package encompasses a diverse set of statistical measures, including mean, mode, median, range, quantiles, first quartile (25th percentile), third quartile (75th percentile), as well as frequency tables (both cumulative and relative), standard deviation (applicable to both populations and samples), and variance (once again, for populations and samples).
```
Comment (roberto-butti) [inline README.md:42]: please change to this:
The functions provided by this package, cover a range of measures, including mean, mode, median, range, quantiles, first quartile (25th percentile), third quartile (75th percentile), frequency tables  (both cumulative and relative), standard deviation (applicable to both populations and samples), and variance (once again, for populations and samples).

**Review by roberto-butti — CHANGES_REQUESTED** [review]

**Comment by shraddha761** [comment]
@roberto-butti done with changes

**Review by roberto-butti — APPROVED** [review]

**Pull request merged**